models/RecruiterModel.js

```diff
@@ -0,0 +1,59 @@
+const mongoose = require("mongoose");
+
+const Schema = mongoose.Schema;
+const DataTypes = Schema.Types;
+
+const RecruiterSchema = new Schema(
+  {
+    companyName: {
+      type: DataTypes.String,
+      required: true,
+    },
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 2 tabs but found 4 spaces.](https://app.codacy.com/gh/n4alphapesto/hirebook-backend/pullRequest?prid=7771833)
```suggestion
		},
```